src/libs/styles.c

```diff
@@ -434,16 +434,16 @@ static void _export_clicked(GtkWidget *w, gpointer user_data)
 
           if(dt_conf_get_bool("plugins/lighttable/style/ask_before_delete_style"))
           {
-            GtkWidget *dialog_overwrite_export = gtk_dialog_new_with_buttons(_("overwrite style?"), GTK_WINDOW(win), GTK_DIALOG_DESTROY_WITH_PARENT,
+            GtkWidget *dialog_overwrite_export = gtk_dialog_new_with_buttons(_("overwrite file?"), GTK_WINDOW(win), GTK_DIALOG_DESTROY_WITH_PARENT,
```

Comment:
I would keep style here. The fact that it is a file for a style is mostly a details not really important for users.